app/src/main/java/edu/uc/kovaciad/slicetracker/dto/Brand.kt

```diff
@@ -5,11 +5,11 @@ import androidx.room.Entity
 import androidx.room.PrimaryKey
 
 /**
- * @param bid: DB ID for dev use
+ * @param bId: DB ID for dev use
  * @param brandName: User Inputted Name
  */
 @Entity(tableName = "Brand")
-data class Brand(@PrimaryKey val bid: Int,
+data class Brand(@PrimaryKey val bId: Int,
```

Comment:
Made the variable camel case as it is easier to read and follows Kotlin best practices. https://kotlinlang.org/docs/coding-conventions.html#function-names

This is repeated over many files for the id's but I only left the comment on this one.

Reply:
A good idea to align with conventions.  In this case, I'd spell it out entirely.  What is a bId?  I don't know.  But a brandId does make more sense.

As tempting as it is to save keystrokes with abbreviations, you'll find that you waste more time trying to remember which abbreviation you used, than the time you saved on keystrokes with a shorter name.  :)